app/views/meals/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <%= stylesheet_link_tag 'application' %>
+  </head>
+  <body>
+    <header>
+        <div>
+          <p><%= link_to "Go back", meals_path, class: 'go-back' %></p>
+          <%= button_to 'Sign out', destroy_user_session_path, method: :delete, class: 'sign-out' %>
+        </div>
+    </header>
+
+    <%= form_with model: @meal, class: 'new-meal-form', local: true do |form| %>
+      <div>
+        <%= form.text_field :name, placeholder: 'Name of the meal', class: 'input-field' %>
+      </div>
+      <div class="category-input">
+        <%= form.url_field :icon, class: 'input-field', placeholder: 'Put your image URL here' %>
+      </div>
+      <div class="meal-submit">
+        <%= form.submit "Create Meal", class: 'meal-submit-btn' %>
+      </div>
+    <% end %>
```

Comment:
- [ OPTIONAL ] I really appreciate your hard work but I think it would be great if you added an input to let the user choose a category because it is mandatory in the requirements so I recommend adding it as well ✅🙏🏻🔥 but I assigned it as optional because the previous reviewer didn't mention it ✨

![image](https://github.com/ClintonYounge/Meals-Budgeting-App/assets/58553711/fcb4148a-f60b-408d-850a-8d14e7c5ed7a)

Reply:
@Reem-lab Thanks a lot. I was troubled because I don't understand how to implement this. You go into a meal and you place an order for that meal. Selecting the meal (category) or other meals from within the orders of an existing meal would be really confusing to implement. Will try my best. Thanks again.